src/estimation/CMakeLists.txt

```diff
@@ -6,6 +6,8 @@
 # https://www.gnu.org/licenses/old-licenses/lgpl-2.1.html
 # at your option.
 
+project(iDynTree_Estimation CXX)
```

Comment:
Is this necessary? Unless a sub-directory is meant to be built on its own by CMake, there is no reason to but `project`  in  a CMakeLists.txt

Reply:
Removed in https://github.com/robotology/idyntree/pull/783/commits/623493079d6aebd58c8fb58938e2141b7289988d